### ISSUE TITLE
Using Drawtools breaks map tooltips

### ISSUE THREAD

**Issue by StefanBrand** [EOX-A/EOxElements#592]
After I got an error from drawtools when using it with the latest map version, I upgraded drawtools to 0.6.1. Unfortunately, it breaks map tooltips. Here is a screencast that demonstrates the problem:

- drawn feature overlays other layer and breaks the latter's tooltips
- drawn feature is interactive (I don't want it to be interactive)
- drawn feature's tooltip is empty
- after deleting the drawn feature, the empty tooltip still appers where the drawn feature was.

[Peek 2024-01-18 16-09.webm](https://github.com/EOX-A/EOxElements/assets/8948865/6003e774-c5db-4685-84fb-fdb348600fe4)

**Comment by StefanBrand** [comment]
Possibly the problem is that we add a new layer for the drawn feature with a hover interaction.

https://github.com/EOX-A/EOxElements/blob/807ed010fdf248d6cab30952fc42df664a39d3b9/elements/drawtools/src/methods/draw/init-draw-layer.js#L17-L63

 I think that that interaction captures the global tooltip overlay that was added in 3349e996619c059c419e15a087275fa7a7f9aad4

**Comment by StefanBrand** [comment]
Investigation with @RobertOrthofer resulted in a bug fix for the map in #593 .

A workaround on application side helps to mitigate the issue:

```js
      // NOTE: workaround for drawtools capturing tooltip
      this.getEoxMap().selectInteractions["selectHover"].setActive(false);
      this.getEoxMap().selectInteractions["selectClick"].setActive(false);
```

Closing this and opening an issue to investigate a more future-proof solution for drawtools.
